src/main/java/com/digirati/taxonomy/manager/lookup/TextLookupService.java

```diff
@@ -0,0 +1,72 @@
+package com.digirati.taxonomy.manager.lookup;
+
+import com.digirati.taxonomy.manager.lookup.model.Word;
+import com.digirati.taxonomy.manager.lookup.model.LookupResultContext;
+import com.digirati.taxonomy.manager.lookup.model.ConceptMatch;
+import com.digirati.taxonomy.manager.lookup.model.TermMatch;
+import com.digirati.taxonomy.manager.lookup.normalisation.TextNormaliser;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/** Service to orchestrate all the steps in looking up concepts from a piece of input text. */
```

Comment:
The style guide disagrees: https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line.

What is the personal preference? I personally prefer folding these comments into one line, to avoid adding vertical padding.

Reply:
Personal preference is to take a newline after the opening tag, but I don't care enough to make a big deal about it. As long as it's as-per the style guide.

Reply:
My personal preference on this would be to have the new line - that's how I wrote it originally but it was removed by formatter so I figured it would be best to leave it without for now